ui/admin/app/components/model-providers/ModelProviderForm.tsx

```diff
@@ -1,9 +1,12 @@
 import { zodResolver } from "@hookform/resolvers/zod";
+import { Link } from "@remix-run/react";
```

Comment:
So this is super confusing (It's on my list to make it easier) but we have a `Link` component found in `~/components/ui/link.tsx` that wraps this and applies some extra stuff to it. (Primarily the ability to disguise it as a button)

Reply:
All good! Saw the remix-run was the most used so went with a majority guess 😂 Thank you for the heads up, will use that from now on!

Reply:
Yeah I have a bad habbit of getting implementing a new idea and not going back and updating previous ones. You did the right thing here :)